src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -43,7 +43,7 @@ public class ConsensusManager : IConsensusManager
         /// <summary>The amount of blocks from consensus the node is considered to be synced.</summary>
         private const int ConsensusIsConsideredToBeSyncedMargin = 5;
 
-        private readonly Network network;
+        public Network Network { get; private set; }
```

Comment:
please make it private. 
Instead of making it public to use in CMB it will probably be better to just provide network to the constructor of CMB

Reply:
agree

Reply:
done